examples/fundamentals/data_container.myst.md

```diff
@@ -99,20 +97,20 @@ Furthermore, inside `idata`, PyMC has automatically saved the observed (endogeno
 idata.observed_data
 ```
 
-In this next model, we create a `pm.ConstantData` containers to hold the observations, and pass this container to the `observed`. We also make a `pm.ConstantData` container to hold the `x` data:
+In this next model, we create a `pm.Data` container to hold the observations, and pass this container to the `observed`. We also make a `pm.Data` container to hold the `x` data:
 
 ```{code-cell} ipython3
 with pm.Model() as no_data_model:
-    x_data = pm.ConstantData("x_data", x)
-    y_data = pm.ConstantData("y_data", y)
+    x_data = pm.Data("x_data", x)
+    y_data = pm.Data("y_data", y)
     beta = pm.Normal("beta")
     mu = pm.Deterministic("mu", beta * x_data)
     sigma = pm.Exponential("sigma", 1)
     obs = pm.Normal("obs", mu=mu, sigma=sigma, observed=y_data)
     idata = pm.sample(random_seed=RANDOM_SEED)
 ```
 
-Because we used a `pm.ConstantData` container, the data now appears on our probabilistic graph. It is downstream from `obs` (since the `obs` variable "causes" the data), shaded in gray (because it is observed), and has a special rounded square shape to emphasize that it is data. We also see that `x_data` has been added to the graph.
+Because we used a `pm.Data` container, the data now appears on our probabilistic graph. It is downstream from `obs` (since the `obs` variable "causes" the data), shaded in gray (because it is observed), and has a special rounded square shape to emphasize that it is data. We also see that `x_data` has been added to the graph.
```

Comment:
```suggestion
Because we used a `pm.Data` container, the data now appears in our probabilistic graph. It is downstream from `obs` (since the `obs` variable "causes" the data), shaded in gray (because it is observed), and has a special rounded square shape to emphasize that it is data. We also see that `x_data` has been added to the graph.
```

Reply:
Yup, I missed that. Thank you, for catching it. I will fix it.